README.md

```diff
@@ -50,6 +50,20 @@ fs.createReadStream('data.csv')
 
 Note that `through2.obj(fn)` is a convenience wrapper around `through2({ objectMode: true }, fn)`.
 
+## Do you need this?
+
+Since Node.js introduced [Simplified Stream Construction](https://nodejs.org/api/stream.html#stream_simplified_construction), many uses of **through2** have become redundant. Consider whether you really need to use **through2** or just want to use the core `'stream'` package (or ideally the `'readable-stream'` package from npm):
+
+```js
+const { Transform } = require('stream')
```

Comment:
In the documentation I would welcome to see `readable-stream` instead of the nodejs `stream` API and make `stream` the "if you really need to"-option.